stage0/run.go

```diff
@@ -337,6 +337,18 @@ func Run(cfg RunConfig, dir string) {
 		}
 	}
 
+	destRootfs := common.Stage1RootfsPath(dir)
+	flavor, err := os.Readlink(filepath.Join(destRootfs, "flavor"))
+	if err != nil {
+		log.Fatalf("Error reading flavor: %v", err)
```

Comment:
Nothing else here capitalizes the log.Faltalf() errors, just be consistent for now, even if they should all be capitalized.